cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -19,6 +22,12 @@
 #include "i915_private_android.h"
 #endif
 
+#define IVSH_WIDTH 1920
+#define IVSH_HEIGHT 1080
+
+struct driver *cros_gralloc_driver::drv_ivshmem_ = nullptr;
+int32_t cros_gralloc_driver::drv_num = 0;
```

Comment:
most class member are initialized in class definition, is there any special requirement to do it separately?